projects/plugins/jetpack/extensions/blocks/subscriptions/subscriptions.php

```diff
@@ -173,7 +173,7 @@ function fetch_subscriber_counts() {
 	$subs_count = 0;
 	if ( is_wpcom() ) {
 		$subs_count = array(
-			'value' => \wpcom_fetch_subs_counts( false ),
+			'value' => \wpcom_fetch_subs_counts( true ),
```

Comment:
Do we need to worry about the potential impact on performance this change may have on WordPress.com?